### PR TITLE
Add Update-python-CIConfig

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Move `UpdateParamsJsonPython` from `https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/update-docs-ci.ps1` to LanguageSettings.ps1.
- Rename function to `Update-python-CIConfig`
- Add default `$monikerId=$null` argument to help function reuse across the languages.
- Switch from `BuildNumber` to `BuildID` for test release version.

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -117,4 +117,62 @@ function Get-python-GithubIoDocIndex() {
...
+      }
+    }
+    else {
+      $newItem = New-Object PSObject -Property @{ 
+          package_info = New-Object PSObject -Property @{ 
+            prefer_source_distribution = "true"
+            install_type = "pypi"
+            name=$releasingPkg.PackageId
+          }
+          excludePath = @("test*","example*","sample*","doc*")
```
Comment (scbedd) [inline eng/scripts/Language-Settings.ps1:169]: While you're updating this. Can you change this to `exclude_path` instead of `excludePath`?

Comment (scbedd) [reply eng/scripts/Language-Settings.ps1:169]: [PR for Ref](https://github.com/Azure/azure-sdk-tools/pull/1196)

**Review by scbedd — APPROVED** [review]
Please test this with at least a local run of the script as the release would actually execute. This _looks_ fine but it's easy to miss run-time bugs.

The plan is to merge the actual update to the call of this new function in the common `docs-ci` script right?

**Review by scbedd — APPROVED** [review]

**Review by weshaggard — APPROVED** [review]
Changes looks good.

**Pull request merged**